### PR TITLE
Preserve data inside the web browser

### PR DESCRIPTION
**Pull request by katarighe**
In this activity, the completed data form of the contact us page is saved inside the local storage of the browser. The purpose of this task is that it allows the user reloads the page that the data they putted in is preserved. It will remain reserved as long as the browser is active. 

**Before**
image

**After** (used with Microsoft Edge)
image

**Walkthrough Video**

https://github.com/katarighe/my-portfolio/assets/79839603/239187b1-a6a0-4495-a318-21d8febb5270

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -29,14 +29,37 @@ const form = document.getElementById('myForm');
...
+
+  const myStorage = {
+    name: document.getElementById('fname').value,
+    email: document.querySelector('.email-input').value,
+    message: document.getElementById('message').value,
+  };
 
   if (email === suggestedEmail) {
+    localStorage.setItem('myStorage', JSON.stringify(myStorage));
     form.submit();
```
Comment (usorfaitheloho) [inline script.js:45]: You did well with this milestone and your effort is highly commendable, please ensure to follow the project requirement judiciously as your form doesn't save data when the page is reloaded which is a major requirement for this milestone. The data inputted in the form should be saved to the local storage and gotten from there after page is reloaded,please do well to implement this for your project 😄 

project requirement :

image

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
> Hi @katarighe  ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

  [ ](https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)
> 
> ## Highlights
> * Introduced local storage to your form ✔️
> * Descriptive pull request ✔️
> * Used GitHub flow ✔️
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by katarighe** [comment]
Update regarding out changes on the portfolio.

By @NewtonOmbese

**Review by Cabraham1 — APPROVED** [review]
Hi  @katarighe , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 Good use of Local Storage✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**